File: WEB/src/Entity/Livraison.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Livraison
 *
 * @ORM\Table(name="livraison")
 * @ORM\Entity
 */
class Livraison
{
    /**
     * @var int
     *
     * @ORM\Column(name="idLivraison", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idlivraison;

    /**
     * @var int
     *
     * @ORM\Column(name="reference", type="integer", nullable=false)
     */
    private $reference;

    /**
     * @var string
     *
     * @ORM\Column(name="dateLivraison", type="string", length=20, nullable=false)
     */
    private $datelivraison;

    /**
     * @var string|null
     *
     * @ORM\Column(name="nomLivreur", type="string", length=20, nullable=true)
     */
    private $nomlivreur;

    /**
     * @var string|null
     *
     * @ORM\Column(name="etat", type="string", length=10, nullable=true)
     */
    private $etat;

    /**
     * @var string|null
     *
     * @ORM\Column(name="nomClient", type="string", length=30, nullable=true)
     */
    private $nomclient;

    public function getIdlivraison(): ?int
    {
        return $this->idlivraison;
    }

    public function getReference(): ?int
    {
        return $this->reference;
    }

    public function setReference(int $reference): self
    {
        $this->reference = $reference;

        return $this;
    }

    public function getDatelivraison(): ?string
    {
        return $this->datelivraison;
    }

    public function setDatelivraison(string $datelivraison): self
    {
        $this->datelivraison = $datelivraison;

        return $this;
    }

    public function getNomlivreur(): ?string
    {
        return $this->nomlivreur;
    }

    public function setNomlivreur(?string $nomlivreur): self
    {
        $this->nomlivreur = $nomlivreur;

        return $this;
    }

    public function getEtat(): ?string
    {
        return $this->etat;
    }

    public function setEtat(?string $etat): self
    {
        $this->etat = $etat;

        return $this;
    }

    public function getNomclient(): ?string
    {
        return $this->nomclient;
    }

    public function setNomclient(?string $nomclient): self
    {
        $this->nomclient = $nomclient;

        return $this;
    }


}

File: Desktop/src/src/entities/Livraison.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entities;

import java.sql.Connection;
import java.sql.Date;
import java.sql.ResultSet;
import java.sql.SQLException;
import java.sql.Statement;
import java.sql.Timestamp;
import javafx.collections.ObservableList;

/**
 *idLivraison	reference	dateLivraison	nomLivreur	etat	nomClient
 * @author dorra
 */
public class Livraison {
  private  int  idLivraison;
  private  int  reference;
  private String  dateLivraison;
  private String  nomLivreur;
  private String  etat;
  private String  nomClient;

    public Livraison() {
    }

    public Livraison(int idLivraison, int reference, String dateLivraison, String nomLivreur, String etat, String nomClient) {
        this.idLivraison = idLivraison;
        this.reference = reference;
        this.dateLivraison = dateLivraison;
        this.nomLivreur = nomLivreur;
        this.etat = etat;
        this.nomClient = nomClient;
    }


  
    public int getIdLivraison() {
        return idLivraison;
    }

    public void setIdLivraison(int idLivraison) {
        this.idLivraison = idLivraison;
    }

    public int getReference() {
        return reference;
    }

    public void setReference(int reference) {
        this.reference = reference;
    }

    public String getDateLivraison() {
        return dateLivraison;
    }

    public void setDateLivraison(String dateLivraison) {
        this.dateLivraison = dateLivraison;
    }

    public String getNomLivreur() {
        return nomLivreur;
    }

    public void setNomLivreur(String nomLivreur) {
        this.nomLivreur = nomLivreur;
    }

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

    public String getNomClient() {
        return nomClient;
    }

    public void setNomClient(String nomClient) {
        this.nomClient = nomClient;
    }

  
                         
}
